src/defaultsettings.cpp

```diff
@@ -286,6 +286,9 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("mono_font_size", font_size_str);
 	settings->setDefault("contentdb_url", "https://content.minetest.net");
 
+	settings->setDefault("update_information_url", "https://api.github.com/repos/minetest/minetest/releases");
```

Comment:
this is a bad idea, it would be better to put a file on minetest.net imo

Reply:
Not redirect, but cache the JSON file.
```
$ curl -i https://api.github.com/repos/minetest/minetest/releases
HTTP/1.1 200 OK
Server: GitHub.com
Date: Wed, 08 Aug 2018 16:24:19 GMT
Content-Type: application/json; charset=utf-8
Content-Length: 92175
Status: 200 OK
X-RateLimit-Limit: 60
X-RateLimit-Remaining: 52  <-- issue is here
X-RateLimit-Reset: 1533746023
```

Reply:
Since the code barely uses any of the 92 KB that JSON file is big, we should definitely host a replacement ourselves.

Reply:
Yeah, agreed. Probably easiest to just add a file directly to https://github.com/minetest/minetest.github.io

Reply:
or we should just use the github API correctly ?

https://developer.github.com/v3/repos/releases/#get-the-latest-release

Reply:
We should not rely on the GitHub API as we may not use it forever. We could use the Github API in Jekyll to automatically make a JSON file with the latest release, this would be future proof